test/tab-size.js

```diff
@@ -0,0 +1,373 @@
+var assert = require('assert');
+var gonzales = require('./../');
+
+var cssStringNonIndented = 'a{color:blue;}';
+var cssStringTabIndented = 'a{\n\tcolor:blue;\n}';
+var lessStringNonIndented = '@color:blue;a{color:@color;}';
+var lessStringTabIndented = '@color:blue;\na{\n\tcolor:@color;\n}';
+var sassStringTabIndented = '$color:blue\na\n\tcolor:$color';
+var scssStringNonIndented = '$color:blue;a{color:$color;}';
+var scssStringTabIndented = '$color:blue;\na{\n\tcolor:$color;\n}';
+
+/**
+ * Returns color declaration start column value.
+ * @param {string} s String to be parsed
+ * @param {Object} [options] Parsing options
+ * @return {number} Start column value
+ */
+function getColorDeclarationStartCol(s, options) {
+    var ast = gonzales.parse(s, options);
+    if(options && options.syntax === 'sass') {
+        return ast
+            .first('ruleset')
+            .last('selector')
+            .first('simpleSelector')
```

Comment:
The idea is to get the value of the start column of the `color` declaration (`color:$color;`) from the following Sass String:
`var sassStringTabIndented = '$color:blue\na\n\tcolor:$color';`